cassiopeia/data/CassiopeiaTree.py

```diff
@@ -186,26 +209,36 @@ def populate_tree(self, tree: Union[str, ete3.Tree, nx.DiGraph]) -> None:
             )
 
     def __check_network_initialized(self) -> None:
+        """Checks that topology has been initialized."""
         if self.__network is None:
             raise CassiopeiaTreeError("Tree has not been initialized.")
 
-    def set_character_matrix(self, character_matrix: pd.DataFrame):
-        """Initializes a character matrix in the object."""
+    def set_character_matrix(
```

Comment:
See response below.

Reply:
I was referring to renaming this function to just `character_matrix` with a `@character_matrix.setter` decorator, which would call the function whenever someone assigns to `tree.character_matrix`, instead of having to call `tree.set_character_matrix()`.

Reply:
Whoops. Wasn't looking at the latest commit. haha